enterprise-suite/tests/smokecommon

```diff
@@ -143,7 +143,7 @@ test_es_console_responding() {
   CONSOLE_URL=$1
   shift
   es_console_responding() {
-      curl_query $CONSOLE_URL "" "$@" | fgrep -q '<title>Enterprise Suite Reactive Console</title>'
+      curl_query $CONSOLE_URL "" "$@" | fgrep -q '<title>Enterprise Suite Lightbend Console</title>'
```

Comment:
Hmm interesting. Why do the ingress tests pass though if this is completely wrong? Is there something messed up in the test detecting failures?

Reply:
"Enterprise Suite Reactive Console" is the correct title I think. I deployed master, and that is the title being used.